internal/cmd/launcher.go

```diff
@@ -76,22 +77,21 @@ func (l *Launcher) Launch() {
 	// if the command does not have dependencies or a custom build
 	if !customBuildRequired(build.Version, deps) {
 		l.gs.Logger.
-			Debug("The current k6 binary already satisfies all the required dependencies," +
-				" it isn't required to provision a new binary.")
+			Debug("Binary provisioning is not required to execute this command.")
 		l.commandExecutor.run(l.gs)
 		return
 	}
 
 	l.gs.Logger.
 		WithField("deps", deps).
-		Info("The current k6 binary doesn't satisfy all the required dependencies, it is required to" +
-			" provision a new binary.")
+		Info("The current k6 binary doesn't satisfy all dependencies, it is required to" +
+			" provision a custom binary.")
```

Comment:
```suggestion
			" provision ad hoc binary.")
```
`custom` sounds too generic term

Reply:
I think this is the term we use in the documentation referring to custom k6 build.